libs/map-editor/src/MapFetcher.ts

```diff
@@ -34,7 +35,7 @@ class MapFetcher {
         stripPrefix: string | undefined
     ): Promise<WAMFileFormat> {
         const result = await this.fetchFile(wamUrl, true, true, internalMapStorageUrl, stripPrefix);
-        const parseResult = WAMFileFormat.safeParse(result.data);
+        const parseResult = WAMFileFormat.safeParse(wamFileMigration.migrate(result.data));
```

Comment:
Shouldn't we do a try catch and throw a LocalUrlError here, if the call to "migrate" fails?